packages/server/api/controllers/reviews.controller.js

```diff
@@ -0,0 +1,42 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getAllReviews = async () => {
+  try {
+    const allReviews = await knex.select().table('reviews');
```

Comment:
When would we need to get all reviews? Wouldn't it always be scoped per product, like the other function does?

Reply:
This function gets **all** reviews, not just the reviews for **one** product (like the other handler in the controller).